### DISCUSSION TITLE
How to fix numpy package issues in Google Colab?

### DISCUSSION THREAD

**Discussion by evmckinney9** [netket/netket#889] [Q&A]
![image](https://user-images.githubusercontent.com/47376937/130973778-fccfec88-065c-455c-a1b3-84cdde1efa18.png)

Hello, hopefully someone more experienced knows how to fix this easily, but I have tried uninstalling and installing other versions of numpy and nothing seems to work.

Thank you.

**Comment by evmckinney9** [comment]
Finally got a combination of pip commands to work right after posting this.

`!pip install numpy --upgrade` then restart notebook,
`!pip install --upgrade netket` then restart notebook again,
then `import netket as nk` will work.

**Reply by PhilipVinc** [reply]
Can you open an issue on numba repository? It would be great if they fixed it

**Reply by evmckinney9** [reply]
I'm not sure that I know what is happening in enough detail to open an issue

**Reply by PhilipVinc** [reply]
Don't worry. Anyway there's already an issue: https://github.com/numba/numba/issues/7339

The numpy that is installed by default in Colab is 1.19, and when netket is installed it pulls numba 0.54 which only works with the older numpy if you install from source...

There are two solutions:

```python
!pip install --upgrade numpy
!pip install netket
# restart
```
or
```
!pip install numba==0.53
!pip install netket
# no need to restart
```

**Reply by PhilipVinc** [reply]
@evmckinney9 Numba has fixed the issue now, and doing `!pip install netket` should work without issue now.

**Comment by PhilipVinc** [comment]
It is because you have the wrong wheel of numba installed wrt to numpy. You need either to downgrade numba or numpy I think..
Il 26 ago 2021, 15:50 +0200, Evan McKinney ***@***.***>, ha scritto:
